install/includes/config_default.php

```diff
@@ -0,0 +1,127 @@
+<?php
```

Comment:
this file should be in manifest folder, and should be named unattended.php

You'll thank me later.

Reply:
I prefer being explained and thanking you now ;)

Reply:
It's a surprise. It will do the same thing (feature wise)

Reply:
You just want it to be gitignored ?

Reply:
@dominic-blain no, justed renamed/moved under  `manifest/unattended.php`

Reply:
Great!